src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -147,10 +148,14 @@ def runner(
                                     (time, command, team),
                                     ...
                                 }
+        :param run_till_end: If true, test runs till the end even if eventually validation passes
+                             If false, test stops once eventually validation passes and fails if time out
```

Comment:
seems like the play test don't work anymore rn (because of the timeout?).

I tried `offense_play_test.py`

Reply:
fixed this, the old test_timeout_s for that test was 15 seconds instead of the default 3